students/matt_casali/Lesson03/mailroom.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+donors = {"William Gates, III": [326892.23, 326892.25], "Mark Zuckerberg": [500.00, 800.00, 2.00], "Jeff Bezos": [877.33],
+          "Paul Allen" : [750.23, 23.53, 999.99], "Dakota Dakota": [10.00, 100.00,1000.00]}
```

Comment:
please note that in this assignment you should be using data structures you have learned so far which are list/tuple this gives you a chance to practice skills using those data structures.

mailroom assignment goes through multiple revisions each time adding new concepts; in lesson 4 it adds dictionary usages.